dockernet/ts-tests/test/main.test.ts

```diff
@@ -0,0 +1,180 @@
+import { Secp256k1HdWallet } from "@cosmjs/amino";
+import { GasPrice, SigningStargateClient } from "@cosmjs/stargate";
+import { fromSeconds } from "@cosmjs/tendermint-rpc";
+import {
+  coinFromString,
+  convertBech32Prefix,
+  decToString,
+  sleep,
+  StrideClient,
+} from "stridejs";
+import { beforeAll, describe, expect, test } from "vitest";
+
+const RPC_ENDPOINT = "http://localhost:26657";
+
+let accounts: {
+  user: StrideClient; // just a normal user account loaded with 100 STRD
+  admin: StrideClient; // the stride admin account loaded with 1000 STRD
+  val1: StrideClient;
+  val2: StrideClient;
+  val3: StrideClient;
+};
+
+// init accounts and wait for chain to start
+beforeAll(async () => {
+  console.log("setting up accounts...");
+
+  const mnemonics: {
+    name: "user" | "admin" | "val1" | "val2" | "val3";
+    mnemonic: string;
+  }[] = [
+    {
+      name: "user",
+      mnemonic:
+        "brief play describe burden half aim soccer carbon hope wait output play vacuum joke energy crucial output mimic cruise brother document rail anger leaf",
+    },
+    {
+      name: "admin",
+      mnemonic:
+        "tone cause tribe this switch near host damage idle fragile antique tail soda alien depth write wool they rapid unfold body scan pledge soft",
+    },
+    {
+      name: "val1",
+      mnemonic:
+        "close soup mirror crew erode defy knock trigger gather eyebrow tent farm gym gloom base lemon sleep weekend rich forget diagram hurt prize fly",
+    },
+    {
+      name: "val2",
+      mnemonic:
+        "turkey miss hurry unable embark hospital kangaroo nuclear outside term toy fall buffalo book opinion such moral meadow wing olive camp sad metal banner",
+    },
+    {
+      name: "val3",
+      mnemonic:
+        "tenant neck ask season exist hill churn rice convince shock modify evidence armor track army street stay light program harvest now settle feed wheat",
+    },
+  ];
+
+  // @ts-expect-error
+  // init accounts as an empty object, then add the accounts in the loop
+  accounts = {};
+  for (const { name, mnemonic } of mnemonics) {
+    // @ts-expect-error
+    // init accounts[name] as an empty object, then add the fields one by one
+    accounts[name] = {};
```

Comment:
is this necessary since you just overwrite it at the end of this for loop?

Reply:
you might be right, this part has gone through a few iterations

Reply:
43bc9815ba8ea998536df5f2059eaf447132df85